CMakeLists.txt

```diff
@@ -63,59 +63,63 @@ find_library(LUA_LIBRARY lua)
 link_libraries(${LUA_LIBRARY})
 
 find_package(Doxygen QUIET)
+include_directories(include)
 
-set(NOVELRT_HEADERS
-  src/NovelWorldObject.h
-  src/GeoBounds.h
-  src/GeoVector.h
-  src/GraphicsCharacterRenderData.h
-  src/KeyCode.h
-  src/Lazy.h
-  src/NovelAudioService.h
-  src/NovelBasicFillRect.h
-  src/NovelBasicInteractionRect.h
-  src/NovelCommonArgs.h
-  src/NovelDebugService.h
-  src/NovelImageRect.h
-  src/NovelInteractionObject.h
-  src/NovelInteractionService.h
-  src/NovelLayeringService.h
-  src/NovelLoggingService.h
-  src/NovelObjectSortComparison.h
-  src/NovelRenderingService.h
-  src/NovelRenderObject.h
-  src/NovelUtilities.h
-  src/NovelRunner.h
-  src/NovelStepTimer.h
-  src/NovelTextRect.h
-  src/RGBAConfig.h
-  src/wrapper.hpp
-  src/NovelCamera.h
-  src/GeoMatrix4.h
-  src/CameraViewChangedEventArgs.h
-  src/ShaderProgram.h
-  src/CameraBlock.h
-  src/NovelRT.h)
-
-set(NOVELRT_SOURCES
-        src/NovelWorldObject.cpp
-  src/GeoBounds.cpp
-  src/NovelAudioService.cpp
-  src/NovelBasicFillRect.cpp
-  src/NovelBasicInteractionRect.cpp
-  src/NovelDebugService.cpp
-  src/NovelImageRect.cpp
-  src/NovelInteractionObject.cpp
-  src/NovelInteractionService.cpp
-  src/NovelLayeringService.cpp
-  src/NovelLoggingService.cpp
-  src/NovelRenderingService.cpp
-  src/NovelRenderObject.cpp
-  src/NovelRunner.cpp
-  src/NovelStepTimer.cpp
-  src/NovelTextRect.cpp
-  src/RGBAConfig.cpp
-  src/NovelCamera.cpp)
+set(NOVELRT_LIBRARY_HEADERS
```

Comment:
Do we want to try and keep these alphabetized?

Reply:
I can reorder this to be alphabetical if you want, I just threw stuff on the end as I added.